### ISSUE TITLE
Leveraging Large (Visual) Language Models for Robot 3D Scene Understanding. (arXiv:2209.05629v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#28466]
Leveraging Large (Visual) Language Models for Robot 3D Scene Understanding. (arXiv:2209.05629v2 [cs.RO] UPDATED)<br>
https://ift.tt/uD3mtsz<br>
<p>Abstract semantic 3D scene understanding is a problem of critical importance
in robotics. As robots still lack the common-sense knowledge about household
objects and locations of an average human, we investigate the use of
pre-trained language models to impart common sense for scene understanding. We
introduce and compare a wide range of scene classification paradigms that
leverage language only (zero-shot, embedding-based, and structured-language) or
vision and language (zero-shot and fine-tuned). We find that the best
approaches in both categories yield $\sim 70\%$ room classification accuracy,
exceeding the performance of pure-vision and graph classifiers. We also find
such methods demonstrate notable generalization and transfer capabilities
stemming from their use of language.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
